gallery/gallery/lib/demos/material/grid_list_demo.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN gridListsDemo
+
+enum GridListDemoType {
```

Comment:
Not really important, but might be better to make this private.

```suggestion
enum _GridListDemoType {
```

Reply:
It's actually used in another file, so I can't make it private.